### DISCUSSION TITLE
Missing utility block commands

### DISCUSSION THREAD

**Discussion by BirminghamBoy** [Zrips/CMI#7397] [Feature Requests]
CMI's utility block commands, such as `/stonecutter`, `/grindstone`, `/workbench`, etc. are great - however, it would be even better if commands for Brewing Stands, Furnaces, Blast Furnaces and Smokers could be added. This would allow CMI to replace plugins such as PortableCraftingInventories fully. Thanks. :)

**Comment by CloudeLecaw** [comment]
i dont belive they will come

https://github.com/Zrips/CMI/discussions/5021
